utility/utility.go

```diff
@@ -31,6 +32,9 @@ func LoggedClose(c io.Closer, errmsg string) {
 }
 
 func LoggedSync(file *os.File, errmsg string) {
+	if viper.GetBool("WALG_DISABLE_FSYNC") {
```

Comment:
Couldn't import `internal.DisableFsyncSetting` because it'd create an import loop

Other option would be to pass the setting as a parameter. Arguably this bool could be passed down through all these calls rather than repeatedly calling `viper.GetBool`

I was also noticing that `LoggedSync` can be redundant, since `writeLocalFile`  is used by `catchup_file_unwrapper.go` which also calls `file.Sync`. So if the default file unwrapper also called `file.Sync` this defer wouldn't be needed in tar_interpreter_new, leaving it only used by `pagefile.go`